### DISCUSSION TITLE
manual refresh does not take effect

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#9581] [General]
Click the refresh button, and the date does not change. In the 8.9.1 version, click the refresh button, and the date changes. I don't know if it's a bug.

![image](https://user-images.githubusercontent.com/17882970/189093114-1ab5b1de-9c5b-41ac-86d1-8d28cc0f61cd.png)

**Comment by wu-sheng** [comment]
Ping @Fine0830 I think this is a bug? I think fresh and auto fresh are both not working.

**Reply by wu-sheng** [reply]
I created an issue to track, https://github.com/apache/skywalking/issues/9582
